charms/istio-pilot/src/charm.py

```diff
@@ -286,64 +294,126 @@ def handle_ingress_auth(self, event):
             )
             return
 
-        t = self.env.get_template('auth_filter.yaml.j2')
-        auth_filters = ''.join(
+        if self.model.config["ssl-crt"] and self.model.config["ssl-key"]:
+            gateway_port = GATEWAY_HTTPS_PORT
+        else:
+            gateway_port = GATEWAY_HTTP_PORT
+
+        t = self.env.get_template("auth_filter.yaml.j2")
```

Comment:
Seems like this comes from another commit. I believe we merged this only in main, but not in 1.11 (?)

Reply:
You're right, this was mangled into the cherry picked code.  I have removed it now